media/css/mozorg/home/home-b.scss

```diff
@@ -0,0 +1,576 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../hubs/buttons';
+
+body {
+    @include open-sans;
+    border-top: 0;
+}
+
+h2,
+h3 {
+    @include zilla-slab;
+}
+
+// hide download button in the nav
+#nav-download-firefox,
+#global-nav-download-firefox {
+    display: none;
+}
+
+// adjust nav spacing to use space freed up by hiding the dl button
+@media #{$mq-tablet} {
+    .moz-global-nav .nav-primary-links {
+        width: calc(100% - 110px);
+    }
+}
+
+@media #{$mq-desktop} {
+    .moz-global-nav .nav-primary-links {
+        width: calc(100% - 220px);
+    }
+}
+
+// flexbox layout for savvy browsers to push nav items to the right
```

Comment:
Is this needed?

Reply:
Ah, now that it's en-US only, nope, no longer needed. Nice spot.